controllers/druid/handler.go

```diff
@@ -272,11 +272,14 @@ func deployDruidCluster(sdk client.Client, m *v1alpha1.Druid) error {
 		}
 	}
 
-	if m.Spec.DeleteOrphanPvc {
-		if err := deleteOrphanPVC(sdk, m); err != nil {
-			e := fmt.Errorf("Error in deleteOrphanPVC due to [%s]", err.Error())
-			sendEvent(sdk, m, v1.EventTypeWarning, "LIST_FAIL", e.Error())
-			logger.Error(e, e.Error(), "name", m.Name, "namespace", m.Namespace)
+	// Ignore on cluster creation
+	if m.Generation > 1 {
+		if m.Spec.DeleteOrphanPvc {
```

Comment:
can be simplified as is m.generation > 1 && m.Spec.DeleteOrphanPvc

Reply:
done